src/main/resources/spell_teleports.tsv

```diff
@@ -0,0 +1,158 @@
+# Destination	Item ID Requirements	Skill Requirements	Quest	Wait	Display Info	Wilderness Level	Varbits
+
+#
+# Standard Spellbook - Varbit 4070=0
+#
+
+# Varrock Teleport
+3213 3424 0	554,556,563;554,563,4696;556,563,4699;563,4697;556,563,4694;554,563,4695;554,563,1381;554,563,20730;554,563,20736;556,563,1387;556,563,11787;556,563,3053;563,11998;556,563,20714;563,1381,20714;563,20730,20714;563,20736,20714	25 Magic		4	Varrock Teleport	20	4070=0
```

Comment:
There are a lot of combinations of runes/staffs that allow you to teleport. I made a python script to parse the HTML table from the [wiki page](https://oldschool.runescape.wiki/w/Varrock_Teleport) of each teleport to get all the combinations. Then just converted the item names to ids.